packages/mongodb-chatbot-eval/src/evaluate/checkResponseQuality.ts

```diff
@@ -0,0 +1,173 @@
+import { stripIndents } from "common-tags";
+import e from "express";
+import {
+  FunctionDefinition,
+  OpenAIClient,
+  OpenAiChatMessage,
+} from "mongodb-chatbot-server";
+
+export interface CheckResponseQualityParams {
+  received: string;
+  expectedOutputDescription: string;
+  openAiClient: OpenAIClient;
+  deploymentName: string;
+  fewShotExamples?: ResponseQualityExample[];
+}
+
+export interface ResponseQualityExample {
+  messages: OpenAiChatMessage[];
+  expectation: string;
+  llmOutput: CheckResponseQuality;
+}
+
+export interface CheckResponseQuality {
+  reason: string;
+  meetsChatQualityStandards: boolean;
+}
+
+export async function checkResponseQuality({
+  received,
+  expectedOutputDescription,
+  openAiClient,
+  deploymentName,
+  fewShotExamples,
+}: CheckResponseQualityParams): Promise<CheckResponseQuality> {
+  const userMessageContent = stripIndents`
+
+  <Content>
+
+  ${received}
+
+  <End of content>
+
+  <Expectation>
+  ${expectedOutputDescription}
+  <End of expectation>
```

Comment:
very nitty - why extra newlines around the `received` content but not `expectedOutputDescription`?

Reply:
good catch removing above. lets save a few precious tokens